collections/specprocessor/crowdsource/controlpanel.php

```diff
@@ -1,6 +1,8 @@
 <?php
 include_once('../../../config/symbini.php');
 include_once($SERVER_ROOT.'/classes/OccurrenceCrowdSource.php');
+if($LANG_TAG != 'en' && file_exists($SERVER_ROOT.'/content/lang/collections/specprocessor/crowdsource/controlpanel.'.$LANG_TAG.'.php')) include_once($SERVER_ROOT.'/content/lang/collections/specprocessor/crowdsource/controlpanel.'.$LANG_TAG.'.php');
+else include_once($SERVER_ROOT.'/content/lang/collections/specprocessor/crowdsource/controlpanel.en.php');
```

Comment:
I'm not sure that this is necessary.

Assuming LANG_TAG is truthy, 
`include_once($SERVER_ROOT.'/content/lang/collections/specprocessor/crowdsource/controlpanel.'.$LANG_TAG.'.php');` would cover both the en case and the non-en case, right?

Reply:
It would, but it's possible that the .es or .pt or whatever else it is doesn't exist (it hasn't been translated yet), so in that case, you would have NO labels at all...which could be bad!

Reply:
Ok. So we're assuming that the "*.en.php" version always exists. Got it.